usl_pipeline/README.md

```diff
@@ -2,6 +2,8 @@ Getting Started
 ===============
 
 To install the package and its requirements for local development:
+- System dependencies:
+  - GDAL version 3.6.4 or later (apt-get install gdal-bin libgdal-dev)
```

Comment:
Should folks be using the ubuntugis/ppa you're using in the CI process?

Reply:
I guess it might depend on the OS. I don't know what it needs to be in case of windows. Actually https://gdal.org/download.html gives a lot of ways to download binaries for all the recent versions. I think it's also possible (in theory) to compile GDAL from sources.

Reply:
I decided to point to https://gdal.org/download.html link in the readme instead of "apt-get" command.